addons/zylann.hterrain/shaders/multisplat16.shader

```diff
@@ -52,13 +55,16 @@ vec4 pack_normal(vec3 n, float a) {
 // Blends weights according to the bump of detail textures,
 // so for example it allows to have sand fill the gaps between pebbles
 vec4 get_depth_blended_weights(vec4 splat, vec4 bumps) {
-	float dh = 0.2;
-
+	float dh = height_boost;
+	bumps.r = 1.0 - pow(1.0 - bumps.r, 0.5);
```

Comment:
Careful with `pow`, it's a transcendental math function, it's expensive. Here you raise to power 0.5, which is actually square root, so maybe use `sqrt` instead?
Not sure what this formula does